06.bitstream.syntax.md

```diff
@@ -8,47 +8,14 @@ each of the syntax elements is presented in [Section 6][].
 ### AOMedia ITU-T T.35 metadata syntax
 
 | --------------------------------------------------------- | ---------------- |
-| metadata_itut_t35( ) {                                    | **Type**
+| metadata_aom_itu_t_t35( ) {                                    | **Type**
 |     @@itu_t_t35_country_code                              | f(8)
-|     if ( itu_t_t35_country_code == 0xB5 ) {
-|         @@itu_t_t35_terminal_provider_code                | f(16)
-|         if ( itu_t_t35_terminal_provider_code == 0x5890 ) {
-|             aom_itu_t_t35_payload()
-|     }
-| }
-{:.syntax }
-
-**Note:** The last byte of the valid content of the data is
-considered to be the last byte that is not equal to zero. This rule is to
-prevent the dropping of valid bytes by systems that interpret trailing zero
-bytes as a padding continuation of the trailing bits.
-This implies that when any payload data is present,
-at least one byte of the payload data (including the trailing bit) shall not be equal to 0.
-{:.alert .alert-info }
-
-
-### AOMedia ITU-T T.35 payload syntax
-
-| --------------------------------------------------------- | ---------------- |
-|aom_itu_t_t35_payload( ) {                                 | **Type**
+|     @@itu_t_t35_terminal_provider_code                    | f(16)
 |     @@itu_t_t35_terminal_provider_oriented_code           | f(8)
-|     if ( itu_t_t35_terminal_provider_oriented_code == 0x01 ) {
```

Comment:
I think it would be better to still keep the conditional statement here (even if in the context of this spec this will always be 0x01). My understanding is that the definition of `metadata_aom_itu_t_t35` will move out to the [AOMedia metadata registry](https://aomediacodec.github.io/metadata-registry/) specification. Therefore, it would be better to keep the if statement to avoid confusion. Maybe we can keep the if and also add a comment after if saying that other values of the `itu_t_t35_terminal_provider_oriented_code` are out of scope in this specification?